deploy/olm-catalog/jaeger.clusterserviceversion.yaml

```diff
@@ -63,21 +63,42 @@ metadata:
         }
       ]
 spec:
-  displayName: jaeger-operator
+  displayName: Community Jaeger Operator
   description: |-
     Jaeger, inspired by [Dapper](https://research.google.com/pubs/pub36356.html) and [OpenZipkin](http://zipkin.io/), is a distributed tracing system released as open source by Uber Technologies. It is used for monitoring and troubleshooting microservices-based distributed systems.
 
-    ### Supported Features
+    ### Core capabilities
+
+    Jaeger is used for monitoring and troubleshooting microservices-based distributed systems, including:
+
+    * Distributed context propagation
+    * Distributed transaction monitoring
+    * Root cause analysis
+    * Service dependency analysis
+    * Performance / latency optimization
+    * OpenTracing compatible data model
+    * Multiple storage backends: Cassandra, Elasticsearch, memory.
+
+    ### Operator features
 
     * **Multiple modes** - Supports `allInOne`, `production`, and `streaming` [modes of deployment](https://github.com/jaegertracing/jaeger-operator#strategies).
 
     * **Configuration** - Directly pass down all supported Jaeger configuration through the Operator.
 
-    * **Storage** - Configure storage used by Jaeger. By default, `memory` is used.
+    * **Storage** - Configure storage used by Jaeger. By default, `memory` is used. Other options include `elasticsearch` or `cassandra`. On openshift, the operator can delegate creation of an Elasticsearch cluster to the Elasticsearch Operator if deployed.
```

Comment:
Since this is the community template, should the reference to OpenShift be to OKD instead since that is the upstream community?  (If not, then camelcase OpenShift please.)

Reply:
Want to minimise the differences between the CSVs, so will use `OpenShift` as a general term.